client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -3,13 +3,21 @@ import { Input } from "antd";
 import styled from "styled-components";
 import Head from "./Head";
 import { Section } from "../StyledModal";
+import { theme } from "constants/theme";
+const { black } = theme.colors;
 
-const TitleInput = styled(Input)``;
+const TitleInput = styled(Input)`
+  font-size: 1.5rem !important;
+  color: ${black} !important;
+  border: solid 0.1rem;
+`;
 const { TextArea } = Input;
 const TextInput = styled(TextArea)`
   border-top: none;
+  font-size: 1.2rem !important;
```

Comment:
Avoid the important. Try a rule like `.ant-modal-root .ant-modal-content .ant.modal-body & {`

Reply:
@silenceway I tried and it wound't work. What do you advise? 
<img width="266" alt="Screen Shot 2020-07-11 at 9 42 55 PM" src="https://user-images.githubusercontent.com/49251966/87239061-7d810a00-c3bf-11ea-9d5f-9277af41dd0a.png">

Reply:
I think is missing the `&` or copy the rule as I wrote it.

Reply:
It still wont allow me to change the field.
<img width="444" alt="Screen Shot 2020-07-12 at 12 15 20 AM" src="https://user-images.githubusercontent.com/49251966/87241123-c7c0b600-c3d4-11ea-94c7-a8c38d02031b.png">

Reply:
That screenshot has a double point, wrong rule.

Reply:
Im not sure why this is not working. I tired all possibilities. :( Have you by any chance tried it and it worked at your end? 
<img width="483" alt="Screen Shot 2020-07-12 at 12 34 16 PM" src="https://user-images.githubusercontent.com/49251966/87255023-02aa0480-c43c-11ea-9323-193dc4617147.png">